src/pages/Defi/components/OpportunityCard.tsx

```diff
@@ -88,26 +90,44 @@ export const OpportunityCard = ({
 
   if (!asset) return null
 
+  const getOpportunityName = () => {
+    if (assetId === foxEthLpAssetId) return 'ETH-FOX UNI V2'
```

Comment:
we're using this magic string in a lot of places, can we pop in it constants plz?

similarly the same concept of a map here applies for `assetId`s with manual name overrides